include/wx/osx/iphone/private.h

```diff
@@ -225,13 +230,12 @@ protected :
 
     @end
 
-    @interface wxUIView : UIView
+    @interface wxUIView : UIScrollView
     {
     }
 
     @end // wxUIView
 
-
```

Comment:
It would be great to avoid whitespace-only changed intermingled with significant ones please.